src/ngraph/node.hpp

```diff
@@ -79,11 +80,19 @@ namespace ngraph
     /// Alias useful for cloning
     using NodeMap = std::unordered_map<ngraph::Node*, std::shared_ptr<ngraph::Node>>;
 
+    struct NodeTypeInfo
+    {
+        const char* name;
```

Comment:
Why do you use char* and not std::string?

Reply:
std::string cannot be a constexpr so initialization happens in unpredictable ways.

Reply:
Also, `description()` will return a string.